test/findLonest.test.js

```diff
@@ -0,0 +1,14 @@
+import assert from 'assert';
+import last from '../findLongest';
+
+describe('last', function () {
+
+  it('should return the longest element', function () {
+    var array = ['abc', 'abcd', 'abcde'];
+    assert.strictEqual(last(array), 'abcde');
+  });
+  it('should return the longest element', function () {
+    var array = [];
+    assert.strictEqual(last(array), '');
+  });
```

Comment:
A test for handling null and undefined should also be added for null saftey check.

Reply:
I fixed it